AppInspector/rules/default/frameworks/rust.json

```diff
@@ -0,0 +1,82 @@
+﻿[
+  {
+    "name": "Development Framework: Rocket",
+    "id": "AI025610",
+    "description": "Development Framework: Rocket",
+    "applies_to": [ "rust" ],
+    "tags": [ "Framework.Development.Rocket" ],
+    "severity": "moderate",
+    "patterns": [
+      {
+        "pattern": "extern crate rocket;|use rocket",
+        "type": "regex",
+        "scopes": [ "code" ],
+        "modifiers": [ "i" ]
```

Comment:
I don't think you need these `i` modifiers -- these should be case sensitive.

Reply:
copy paste -will do